kernel/virtio/virtio_blk.c

```diff
@@ -178,7 +200,6 @@ int handle_virtio_blk_interrupt(void *arg __attribute__((unused)))
     if (blk_configured) {
```

Comment:
Do we need to register a block interrupt handler at all? `virtio_config_block()` sets `VIRTQ_AVAIL_F_NO_INTERRUPT` so this handler should never get called.